api_spec.md

```diff
@@ -61,11 +61,19 @@ In the case of a multi-frame annotation job, an array of URLs may be given. Note
 
 *string* -- This is intended to be a unique ID for the user performing annotations. It will be assigned to each annotation that this user creates during the session.
 
-### `on_submit`
+### `submit_buttons`
```

Comment:
higher up in this file (~line 35), the `on_submit` field in the constructor should be changed to `submit_buttons` and the typehints should be updated